bot.js

```diff
@@ -76,12 +76,12 @@ function isItGoodNight(msg) {
 function reactWithCrown(msg) {
   msg.react("👑");
 }
-
+// this is a function for a boy
 function IAmBoy(msg) {
   msg.member.roles.add("797567992007360522");
   msg.reply("Role 'Baylar' Succesfully Added!");
 }
-
+//this is a function for a girl
```

Comment:
I dont like this comment.

Reply:
I understand.. I will change the comment.